### ISSUE TITLE
Quest objective tracker "Collections" submenu randomly loses the minimize arrow with the textures removed

### ISSUE THREAD

**Issue by radhaz** [zstenger93/zUI#32]
I can't create a situation that consistently recreates the problem so I know it's going to be a PITA to investigate; however, as the title states the arrow to the right of "Collections" will randomly vanish. Performing a reload fixes the issue every time and it's a very minor inconvenience. The overall objective tracker panels collapse/minimize never goes away just the Collections submenu.

**Comment by zstenger93** [comment]
@radhaz 
Can you tell me, how can I bring up the collections tab aka what I need to set to track?
I think I've never used that part. However, there are elements of the objective tracker that are not covered yet because at the moment I couldn't find a way to access them.
I will take a look at it.

**Comment by zstenger93** [comment]
I checked it, it's kinda sus. I have no idea right now what on earth is causing that. Seems way too random

**Comment by radhaz** [comment]
> I checked it, it's kinda sus. I have no idea right now what on earth is causing that. Seems way too random

Yeah as far bugs go it's extremely random and when it does occur it has negligible impact but I felt like I should report the bug regardless.

**Comment by zstenger93** [comment]
> > I checked it, it's kinda sus. I have no idea right now what on earth is causing that. Seems way too random
> 
> Yeah as far bugs go it's extremely random and when it does occur it has negligible impact but I felt like I should report the bug regardless.

Of course, no worries. If you find anything else, just let me know!
The funniest that sometimes it's just disappears with the title as well, but then a while after reappears.
I bet it's playing hide & seek
